python/doc/usecases/use_case_flood_model.rst

```diff
@@ -11,59 +11,59 @@ The following figure presents a dyke protecting industrial facilities.
 When the river level exceeds the dyke height, flooding occurs.
 The model is based on a crude simplification of the 1D hydrodynamical equations of Saint-Venant
 under the assumptions of uniform and constant flow rate and large rectangular sections.
+This model was first introduced in [deRocquigny2006]_, and then used in [Limbourg2010]_,
+[deRocquigny2012]_, [iooss2015]_, [baudin2015]_.
 
 .. figure:: ../_static/flooding_section.png
     :align: center
     :alt: flooding section
     :width: 50%
 
-    Flooding section
+    **Figure 1.** Flooding section.
 
 
-Four independent random variables are considered:
+Height independent random variables are considered:
 
-  - :math:`Q`: flow rate :math:`[m^3 s^{-1}]`
-  - :math:`K_s`: Strickler :math:`[m^{1/3} s^{-1}]`
-  - :math:`Z_v`: downstream height :math:`[m]`
-  - :math:`Z_m`: upstream height :math:`[m]`
+  - :math:`Q`: flow rate :math:`[m^3 s^{-1}]`;
+  - :math:`K_s`: Strickler :math:`[m^{1/3} s^{-1}]`;
+  - :math:`Z_v`: downstream height :math:`[m]`;
+  - :math:`Z_m`: upstream height :math:`[m]`;
+  - :math:`B`: river width :math:`[m]`;
+  - :math:`L`: river length :math:`[m]`;
+  - :math:`Z_b`: altitude of the river banks :math:`[m]`;
+  - :math:`H_d`: height of the dyke :math:`[m]`.
 
 When the Strickler coefficient increases, the riverbed generates less friction.
 
-The model depends on four parameters:
-
- * the height of the dyke: :math:`H_d = 3` :math:`[m]`,
- * the altitude of the river banks: :math:`Z_b = 55.5` :math:`[m]`,
- * the river length: :math:`L = 5000` :math:`[m]`,
- * the river width: :math:`B = 300` :math:`[m]`.
-
 The altitude of the dyke is:
 
- .. math::
+.. math::
     Z_d = Z_b + H_d
 
 
 The slope :math:`\alpha` of the river is assumed to be close to zero, which implies:
 
- .. math::
+.. math::
     \alpha = \frac{Z_m - Z_v}{L},
 
 if :math:`Z_m \geq Z_v`.
-The water depth is:
+The water depth is ([deRocquigny2012]_ eq. 3.2 page 79) :
 
- .. math::
+.. math::
     H = \left(\frac{Q}{K_s B \sqrt{\alpha}}\right)^{0.6},
 
 
 for any :math:`K_s, Q>0`.
 The flood altitude is:
 
- .. math::
+.. math::
     Z_c = H + Z_v.
 
 
-The altitude of the surface of the water is greater than the altitude of the top of the dyke (i.e. there is a flood) if:
+The altitude of the surface of the water is greater than the altitude of
+the top of the dyke (i.e. there is a flood) if ([deRocquigny2012]_ eq. 3.3 page 79):
```

Comment:
put i.e. in italic?

Reply:
I am not sure : [this ref.](https://www.ncbi.nlm.nih.gov/books/NBK995/#_A291_) and [this](https://www.quora.com/Should-I-italicize-i-e-and-e-g-What-do-i-e-and-e-g-stand-for) say we should not italicize. My 80s english book does not say a word about this topic. Do you have a formal evidence?